src/main/java/uk/gov/hmcts/reform/civil/callback/CaseEvent.java

```diff
@@ -251,7 +251,8 @@ public enum CaseEvent {
     NOTIFY_CLAIMANT_TRANSLATED_DOCUMENT_UPLOADED(CAMUNDA),
     NOTIFY_DEFENDANT_TRANSLATED_DOCUMENT_UPLOADED(CAMUNDA),
     NOTIFY_MEDIATION_UNSUCCESSFUL_CLAIMANT_LR(CAMUNDA),
-    NOTIFY_MEDIATION_UNSUCCESSFUL_DEFENDANT_LIP(CAMUNDA);
+    NOTIFY_MEDIATION_UNSUCCESSFUL_DEFENDANT_LIP(CAMUNDA),
+    SEND_SDO_ORDER_TO_LiP_DEFENDANT(CAMUNDA);
```

Comment:
should be capital LIP look at example above

Reply:
done.